city_scrapers/spiders/chi_ssa_35.py

```diff
@@ -0,0 +1,122 @@
+from datetime import datetime
+from io import BytesIO, StringIO
+
+import requests
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from pdfminer.high_level import extract_text_to_fp
+from pdfminer.layout import LAParams
+
+
+class ChiSsa35Spider(CityScrapersSpider):
+    name = "chi_ssa_35"
+    agency = "Chicago Special Service Area #35 Lincoln Ave"
+    timezone = "America/Chicago"
+    start_urls = [
+        "https://www.lincolnparkchamber.com/"
+        + "businesses/special-service-areas/lincoln-avenue-ssa/ssa-administration/"
+    ]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        content_div = response.css("div.content_block.content.background_white")
+        _ = content_div.css("h4::text").getall()
```

Comment:
Why are you ignoring this variable?

Reply:
I was using it for the commission titles in case i needed it later, but since they were all the same i just ignore it.

Reply:
This can be removed if it isn't being used

Reply:
Ok, i'll remove